atlp-rwanda/ecommerce-app-predators-fe
# #60 chore: redux setup

What does this PR do?
This pull request implements the setup of Redux in our client application.

Description of Task to be completed
1. Installed Redux and Redux Toolkit to manage the application state.
2. Created Redux store with the necessary configuration.
3. Defined actions and reducers to handle the application state.
4. Connected Redux store to the application components using the `useSelector` and `useDispatch` hooks.
5. Updated the relevant components to use Redux for state management.

How should this be manually tested?
1. Run the application and perform actions that trigger state changes.
2. Verify that the application behaves correctly and the state is updated as expected.
3. Test different scenarios to ensure the Redux store is working correctly.

Screenshots (if appropriate)
N/A
